templates/Prometheus/0/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+ranch-eye:
+  ports:
+  - 9104:9104/tcp
+  labels:
+    io.rancher.scheduler.global: 'true'
+  tty: true
+  image: rucknar/ranch-eye:01
+  stdin_open: true
+  net: "host"
+
+node-exporter:
+  ports:
+    - 9100:9100
+  labels:
+    io.rancher.scheduler.global: 'true'
+  tty: true
+  image: prom/node-exporter:latest
+  stdin_open: true
+  net: "host"
+
+prom-conf:
+    tty: true
+    labels:
+      io.rancher.container.pull_image: always
+    image: rucknar/prom-conf:02
+    command: /bin/sh -c "/usr//bin/confd -onetime -backend rancher -prefix /latest && cat"
```

Comment:
Its discouraged to use latest prefix for anything being deployed. Best to pin it, in this case /2015-12-19